src/serial.rs

```diff
@@ -156,6 +167,19 @@ usart_pins! {
         rx => [gpioa::PA5<Alternate<AF5>>, gpioc::PC1<Alternate<AF2>>],
     }
 }
+// TODO: The ST SVD files are missing the entire PE enable register.
+//       Re-enable as soon as this gets fixed.
+// #[cfg(feature = "stm32f091")]
+// usart_pins! {
+//     USART5 => {
+//         tx => [gpioe::PE10<Alternate<AF1>>],
+//         rx => [gpioe::PE11<Alternate<AF1>>],
+//     }
+//     USART6 => {
+//         tx => [gpiof::PF9<Alternate<AF1>>],
+//         rx => [gpiof::PF10<Alternate<AF1>>],
+//     }
```

Comment:
Why is this one commented out given it is GPIOF?

Reply:
Good eye, looks like I accidentally commented out the entire macro call rather than just the `USART5` block. I'm on it.